### PR TITLE
bpo-34245: install Python libraries with more standard 0755 mode

### PR DESCRIPTION
**Pull request by jdemeyer**
https://bugs.python.org/issue34245

### CODE REVIEW

File: Misc/NEWS.d/next/Build/2018-07-27-09-52-48.bpo-34245.bBV0NI.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+The Python library is now installed with write permission (mode 0755), which
+is the standard way of installing libraries.
```
Comment (methane) [inline Misc/NEWS.d/next/Build/2018-07-27-09-52-48.bpo-34245.bBV0NI.rst]: "library" is unclear word.  "py?" files are "Python library" too.
I think "shared module" or "DLL" are bettor wording.

Comment (zooba) [reply Misc/NEWS.d/next/Build/2018-07-27-09-52-48.bpo-34245.bBV0NI.rst]: "Shared library" is probably the correct term. This change does not apply to the Windows DLLs or pyds, and using that term here may be confusing.

**Comment by jdemeyer** [comment]
Changed wording to "shared library"

**Comment by ned-deily** [comment]
I think the change is fine for master (3.8).  As far as backporting, it is hard to make a case that the existing behavior is a bug since the choice of permissions was deliberate.  Since the current behavior has been around for many years with little or no prior complaints, I don't think we should risk breaking anything by changing how the existing maintenance branches behave.

**Pull request merged**